### PR TITLE
add list item using javascript object

### PR DESCRIPTION
**Pull request by Mithi-code**
### Added todo list items dynamically

- Use array for list items

### CODE REVIEW

File: webpack.config.js [hunk 1]
```diff
@@ -13,7 +13,7 @@ module.exports = {
   },
   plugins: [
     new HtmlWebpackPlugin({
-     template: './src/index.html'
+      template: './src/index.html',
     }),
   ],
```
Comment (DeliceLydia) [inline webpack.config.js:14]: - [ ] In this webpack configuration file there is a file called `print.js` am not sure why you are using it here and when I run `npm run build` it is showing an error. I was able to run the server when I removed it

Comment (DeliceLydia) [reply webpack.config.js:14]: - [ ] Please write your README, add some information like a short description etc, and don't forget to add how to get the app locally up and running

**Review by DeliceLydia — CHANGES_REQUESTED** [review]
## Hi @Mithi-code,

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!


## Required Changes ♻️
_Check the comments under the review._



### Cheers and Happy coding!👏👏👏

Feel free to leave a question, or a comment on the PR thread if something is not clear 100%. 

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by AshakaE — APPROVED** [review]
### Hi @Mithi-code   🙋‍♀️, 

# STATUS APPROVED ✔

Your:
   - README 👌 
   
Your project is complete!  There is nothing else to say other than... it's time to merge it 🔗 🚢 
Congratulations! 🎉 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers 🥂  and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**